gen/build_deploy/bash/dcos-launch.spec

```diff
@@ -1,4 +1,12 @@
-a = Analysis(['test_util/launch.py'])
+# Warning:
+#   - hidden import must be used as there is a bug in pyinstaller
+#     https://github.com/pyinstaller/pyinstaller/issues/2185
+#   - data must be decalared explicitly if not a .py file
+#   - Building will suck up the local SSL .so and package it
+#     with the final exe. Ensure build system has OpenSSL 1.0.2g or greater
+a = Analysis(['test_util/launch.py'],
+             hiddenimports=['html.parser'],
+             datas=[('gen/ip-detect/*.sh', 'gen/ip-detect/')])
```

Comment:
does this need the yaml templates that make up the target config gen uses? What is it internally using the ip-detect for?